client/app/dedicatedCloud/dashboard/tiles/options/components/activation-status/activation-status.less

```diff
@@ -0,0 +1,3 @@
+.oui-status {
```

Comment:
Shouldn't rely on the component style instead?

Not sure to understand the benefit of adding this rule here.